docs/api/webview-tag.md

```diff
@@ -606,6 +606,23 @@ listening to the `channel` event with the [`ipcRenderer`](ipc-renderer.md) modul
 See [webContents.send](web-contents.md#contentssendchannel-args) for
 examples.
 
+### `<webview>.sendToFrame(frameId, channel, ...args)`
+
+* `frameId` Integer | [number, number] - the ID of the frame to send to, or a
```

Comment:
I don't love that we're propagating the mistake here that there exists a globally unique frame id that is a single integer. the `| [number, number]` was added to existing APIs to support backwards compatibility and not break apps, but maybe new APIs should only accept the `[number, number]` version and not accept a single number as a frame ID.

Thoughts?

Reply:
we can document it only taking `[number, number]` here, but `sendToFrame` is just forwarded to the webContents, which still takes the single value as well.

Reply:
Alrighty, let's keep it as is then.